### PR TITLE
#27500 Prepare CatalogSearch module Tests for PHPUnit 8

### PR DESCRIPTION
**Pull request by lbajsarowicz**
### Description (*)
Preparing PHPUnit Tests to run with PHPUnit 8.
PRs for each module is separated to make tests easier to merge.

### Fixed Issues (if relevant)
1. magento/magento2#27500

### Manual testing scenarios (*)
N/A

### Questions or comments
PR #27701 needs to be merged first (otherwise PHPUnit tests fail)

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/CatalogSearch/Test/Unit/Model/Adapter/Mysql/Filter/PreprocessorTest.php [hunk 1]
```diff
@@ -1,37 +1,49 @@
...
- * @deprecated
+ * @deprecated Implementation class was replaced
  * @see \Magento\ElasticSearch
  */
-class PreprocessorTest extends \PHPUnit\Framework\TestCase
+class PreprocessorTest extends TestCase
 {
     /**
-     * @var AliasResolver|\PHPUnit_Framework_MockObject_MockObject
+     * @var AliasResolver|\PHPUnit\Framework\MockObject\MockObject
```
Comment (ihor-sviziev) [inline app/code/Magento/CatalogSearch/Test/Unit/Model/Adapter/Mysql/Filter/PreprocessorTest.php]: Could you import it?

**Review by ihor-sviziev — APPROVED** [review]

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Wrong button. Please see my prev. comment

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7344](https://jira.corp.magento.com/browse/ENGCOM-7344) has been created to process this Pull Request

**Comment by lbajsarowicz** [comment]
@ihor-sviziev Done

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-7344](https://jira.corp.magento.com/browse/ENGCOM-7344) has been created to process this Pull Request

**Pull request merged**